scripts/uninstall-zowe.sh

```diff
@@ -28,8 +28,8 @@ SCRIPT_PWD=$(cd $(dirname "$0") && pwd)
 KNOWN_ZOWE_JOB_NAMES="ZOWESVR ZOWESV1 ZOWE1SV ZWE1SV"
 # this should list all known xmem Zowe job names we ever shipped separated by space
 # job name before 1.8.0: ZWESIS01
-# job name preparing for 1.8.0: ZWEXMSTC
-KNOWN_XMEM_JOB_NAMES="ZWESIS01 ZWEXMSTC"
+# job name preparing for 1.8.0: ZWESISTC
+KNOWN_XMEM_JOB_NAMES="ZWESIS01 ZWESISTC"
```

Comment:
@Joe-Winchester @John-A-Davies We need to put all known job names here to make sure no matter what version of previous installation is, the job will be properly stopped.

So the proper change here should be: `KNOWN_XMEM_JOB_NAMES="ZWESIS01 ZWEXMSTC  ZWESISTC"`

Otherwise if previous installation on server started `ZWEXMSTC`, the job won't be stopped and next install test will fail, or reuse the previous Xmem job.

Reply:
Yes, that's right.